Cabal/src/Distribution/Simple/BuildTarget.hs

```diff
@@ -67,7 +67,7 @@ import System.Directory ( doesFileExist, doesDirectoryExist )
 import qualified Data.Map as Map
```

Comment:
[[GH doesn't let me review the commit message so this is here instead]]

Seems this commit doesn't do much of anything anymore the commit message is out of sync here too.

Reply:
Remnants of rebasing, removing changes, adding changes, etc... in the hopes to have some kind of commit history. However, it seems currently better to drop and squash commits that don't do anything anymore.